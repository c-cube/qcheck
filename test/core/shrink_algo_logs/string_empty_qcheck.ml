open QCheck

let strings_are_empty =
  Test.make ~name:"strings are empty"
    string (Log.shrinks_string @@ fun s -> s = "")
;;
try Test.check_exn strings_are_empty with Test.Test_fail _ -> ()
