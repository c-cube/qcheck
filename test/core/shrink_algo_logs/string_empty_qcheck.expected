fails "(Z\139\244"
fails "Z\139\244"
fails "\139\244"
fails "\244"
holds ""
